src/graphql/root/mutation/ln-invoice-fee-probe.ts

```diff
@@ -25,17 +27,19 @@ const LnInvoiceFeeProbeMutation = GT.Field({
       }
     }
 
-    try {
-      const feeSatAmount = await wallet.getLightningFee({
-        invoice: paymentRequest,
-      })
-      // TODO: validate feeSatAmount
-      return {
-        errors: [],
-        amount: feeSatAmount,
-      }
-    } catch (err) {
-      return { errors: [{ message: err.message }] }
+    const feeSatAmount = await lnInvoiceProbeForFee({
+      walletPublicId: walletId,
```

Comment:
doesn't sound right to attach walletPublicId to a walletId because as type they're 2 different things?

Reply:
This is a little strange part of our codebase. We have:

**Domain**
`WalletId`: internal db id
`WalletPublicId`: uuid4 generated wallet id

**GraphQL**
`WalletId`: uuid4 generated wallet id that maps to domain `WalletPublicId`

Reply:
We should probably “fix” it by aligning the same from graphql to domain. Otherwise this will confuse everyone working on our codebase

Reply:
cc @samerbuna @bodymindarts @dolcalmi

Reply:
https://github.com/GaloyMoney/galoy/issues/781

Reply:
Agree with aligning naming as much as possible. WalletPublicId or just `publicId` in domain makes sense.